src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceRadio.vue

```diff
@@ -1,15 +1,35 @@
 <template>
-  <v-radio-group>
-    <v-radio label="Radio One" value="one"></v-radio>
-    <v-radio label="Radio Two" value="two"></v-radio>
-    <v-radio label="Radio Three" value="three"></v-radio>
+  <div>
+    <h3>
+    {{ props.title }}
+  </h3>
+  <br />
+  <v-radio-group :rules="[Rules.requiredRadio()]" @update:model-value="(value) => $emit('update:model-value', value as string)">
```

Comment:
Do we expect every radio to have the same rules? We could add rules to `EceRadioFieldProps` and pass to `v-radio-group`, then we can configure the rules on a per field basis...

If we expect no other rules then OK to hardcode it

Reply:
this is fixed